### ISSUE TITLE
Add Docling in the benchmark

### ISSUE THREAD

**Issue by hongbo-miao** [opendatalab/OmniDocBench#11]
I know MinerU is doing better job than Docling today after I compare the results already.
However, it would be great to add https://github.com/DS4SD/docling in the benchmark : )

**Comment by ouyanglinke** [comment]
Certainly! We will gradually add model evaluation results soon. If you already have inference results from your models, feel free to share them with us.

**Comment by hongbo-miao** [comment]
@ouyanglinke oh Docling cannot recognize formulas, whereas MinerU handles them nearly perfectly. This alone is already a huge win. ☺️

**Comment by ouyanglinke** [comment]
Hi, we evaluate the Docling in OmniDocBench. However, the model infer results looks not quite right for pages in Chinese. Please let us know if there are any issues in the infer code.
If you have no questions about the results, we will update the Docling model results on the evaluation leaderboard soon. 

Here is the result:
 
<table style="width: 92%; border-collapse: collapse; margin: 0 auto;">
  <thead>
    <tr>
      <th rowspan="2">Method Type</th>
      <th rowspan="2">Methods</th>
      <th colspan="2">Text<sup>Edit</sup>↓</th>
      <th colspan="2">Formula<sup>Edit</sup>↓</th>
      <th colspan="2">Formula<sup>CDM</sup>↑</th>
      <th colspan="2">Table<sup>TEDS</sup>↑</th>
      <th colspan="2">Table<sup>Edit</sup>↓</th>
      <th colspan="2">Read Order<sup>Edit</sup>↓</th>
      <th colspan="2">Overall<sup>Edit</sup>↓</th>
    </tr>
    <tr>
      <th>EN</th>
      <th>ZH</th>
      <th>EN</th>
      <th>ZH</th>
      <th>EN</th>
      <th>ZH</th>
      <th>EN</th>
      <th>ZH</th>
      <th>EN</th>
      <th>ZH</th>
      <th>EN</th>
      <th>ZH</th>
      <th>EN</th>
      <th>ZH</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td rowspan="4">Pipeline Tools</td>
    <td>Docling</td>
      <td>0.416</td>
      <td>0.987</td>
      <td>0.999</td>
      <td>1.000</td>
      <td>-</td>
      <td>-</td>
      <td>61.277</td>
      <td>24.972</td>
      <td>0.627</td>
      <td>0.810</td>
      <td>0.313</td>
      <td>0.837</td>
      <td>0.589</td>
      <td>0.908</td>
    </tr>
  </tbody>
</table>


Here is our infer code:

```python
from docling.document_converter import DocumentConverter
import os
from tqdm import tqdm
import pdb

converter = DocumentConverter()

img_path = './OmniDocBench/pdfs'

save_path = './result0106/docling'

for img_name in tqdm(os.listdir(img_path)):
    if not img_name.endswith('.pdf'):
        continue
    
    img_name = img_name.strip()

    save_result_path = os.path.join(save_path, img_name[:-4] + '.md')
    
    if os.path.exists(save_result_path):
        continue

    img_path_tmp = os.path.join(img_path, img_name)
    try:
        result = converter.convert(img_path_tmp)
        result_md = result.document.export_to_markdown()
    except:
        print(img_name)
        continue

    with open(save_result_path, 'w', encoding='utf-8') as output_file:
        output_file.write(result_md)
```

**Comment by hongbo-miao** [comment]
Thanks @ouyanglinke I think it is good start! I don't see any issue. But if anyone find any issues in future, then can always give feedback 😃

**Comment by ouyanglinke** [comment]
Hi, we have updated the Docling result on the evaluation leaderboard.
